src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -174,37 +175,37 @@ CAC.Map.Control = (function ($, L, _) {
             });
         };
 
-        // default to current date and time
-        var now = new Date();
-        // months are zero-based
-        var dateStr = [(now.getMonth() + 1),
-                        now.getDate(),
-                        now.getFullYear()
-                      ].join('-');
+        var formattedTime = when.format('hh:mma');
+        var formattedDate = when.format('MM-DD-YYYY');
 
-        // TODO: implement saving user preferences to fetch for params
         var params = {
             coords: {
-                lat: 39.954688,
-                lng: -75.204677
+                lat: coordsOrigin[0],
+                lng: coordsOrigin[1]
             },
-            mode: ['WALK', 'TRANSIT'],
-            date: dateStr,
-            time: now.toTimeString(),
-            maxTravelTime: 1800,
+            time: formattedTime,
+            date: formattedDate,
+            mode: mode,
+            maxTravelTime: exploreMinutes * 60, // API expects seconds
+            // TODO: add maxWalkDistance as user setting?
             maxWalkDistance: 1609
         };
 
-        locateUser().then(function(data) {
-            params.coords.lat = data[0];
-            params.coords.lng = data[1];
-            getIsochrone(params);
-        }, function(error) {
-            console.log('Could not geolocate user');
-            console.error(error);
-            // use default location
-            getIsochrone(params);
-        });
+        if (coordsOrigin) {
+            deferred.resolve(getIsochrone(params));
```

Comment:
I think this deferred.resolve should be removed (same as how the getIsochrone calls in the next block are called).